project_national/fact_sheets_category_5.yml

```diff
@@ -0,0 +1,593 @@
+
+      - scenario_name: LRMER_LowRECost_2030boxavg
+        type: CO2e
+        elec_folder: data/cambium/2022/LRMER_LowRECost_2030boxavg
+        gas_value: 147.3
+        propane_value: 177.8
+        oil_value: 195.9
+
+      - scenario_name: LRMER_MidCase_2030boxavg
+        type: CO2e
+        elec_folder: data/cambium/2022/LRMER_MidCase_2030boxavg
+        gas_value: 147.3
+        propane_value: 177.8
+        oil_value: 195.9
+
+    utility_bills:
+      - scenario_name: Utility Rates - Fixed + Variable
+        simple_filepath: data/simple_rates/State.tsv
+
+    simulation_output_report:
+      timeseries_frequency: timestep #monthly
+      include_timeseries_total_consumptions: true
+      include_timeseries_fuel_consumptions: true
+      include_timeseries_end_use_consumptions: true
+      include_timeseries_emissions: true
+      include_timeseries_total_loads: false
+
+    reporting_measures:
+      - measure_dir_name: QOIReport
+
+    server_directory_cleanup:
+      retain_in_idf: false
+      retain_schedules_csv: false
+
+### -- Upgrades ---
+# Category 5:  Heat Pumps with Electric Backup
+# To do: costing interpolation from Less for 9.2, 9.5, and 11 HSPF
+upgrades:
+  - upgrade_name: EnergyStar ducted ASHP w/ elec backup - existing elec  
+    # Measure 1
+      #ASHP
+    package_apply_logic: &ducted_ASHP_elec_backup_existing_elec
+      - HVAC Has Ducts|Yes
+      - Heating Fuel|Electricity
+      - or: &not_ASHP_greater_than_SEER_16_9_HSPF
+        - not: HVAC Cooling Efficiency|Ducted Heat Pump
+        - HVAC Heating Efficiency|ASHP, SEER 10, 6.2 HSPF
+        - HVAC Heating Efficiency|ASHP, SEER 13, 7.7 HSPF
+        - HVAC Heating Efficiency|ASHP, SEER 15, 8.5 HSPF
+
+    options:
+
+      - &ducted_ASHP_EnergyStar_elec_backup
+        option: HVAC Heating Efficiency|ASHP, SEER 16, 9.2 HSPF 
+        costs: &costs_ashp_ducted_SEER_16_9-2_HSPF
+        # Source: Custom regression by Brennan Less. Ducted heat pump project costs were regressed on nameplate tons and HSPF,
+        # from data described in in Less et al. https://doi.org/10.20357/B7FP4D. January 2019 $ inflated to April 2023 $ using 1.21 factor.
+        # Ducted ASHP regression n=317. Implicitly includes electrical upgrades and electric backup.
+          - value: 4821.83 # for HSPF 9.2
+            multiplier: Fixed (1)
+          - value: 167.64
+            multiplier: Size, Heating System Primary (kBtu/h)
+        lifetime: &lifetime_ducted_ASHP_elec_backup 15
+
+      - &HP_cooling_efficiency_ducted
+        option: HVAC Cooling Efficiency|Ducted Heat Pump
+
+      - &100_percent_conditioned
+        option: HVAC Cooling Partial Space Conditioning|100% Conditioned
+
+  - upgrade_name: EnergyStar ducted ASHP w/ elec backup - existing natural gas  
+    # Measure 1
+      #ASHP
+    package_apply_logic: &ducted_ASHP_elec_backup_existing_NG
+      - HVAC Has Ducts|Yes
+      - Heating Fuel|Natural Gas
+      - or: *not_ASHP_greater_than_SEER_16_9_HSPF
```

Comment:
I can't decide if this is silly or thorough. No model with a heating fuel of natural gas is going to have a heat pump in baseline. (Same comment for fuel oil and for propane).

Reply:
On thinking it over, I think it is silly. Taking them out.